crypto/src/main/java/org/hyperledger/besu/crypto/KeyPair.java

```diff
@@ -0,0 +1,69 @@
+package org.hyperledger.besu.crypto;
+
+import org.bouncycastle.crypto.params.ECDomainParameters;
+import org.bouncycastle.jcajce.provider.asymmetric.ec.BCECPrivateKey;
+import org.bouncycastle.jcajce.provider.asymmetric.ec.BCECPublicKey;
+
+import java.math.BigInteger;
+import java.security.KeyPairGenerator;
+import java.util.Arrays;
+import java.util.Objects;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class KeyPair {
+
```

Comment:
Should we just use `javax.security.KeyPair` and move the static method elsewhere?

Reply:
Looking through the code that would require a lot of casting from `javax.security.PublicKey` -> `SECPPublicKey`, because `SECPPublicKey.getEncodedBytes` is used in a lot of places and not part of the interface. Considering this I would leave the class for right now, as it is.